src/components/EntityStatus/EntityStatus.js

```diff
@@ -70,20 +67,25 @@ class EntityStatus extends React.Component {
         const {iconPath} = this.props;
 
         return (
-            <ExternalLink target="_blank" href={iconPath}>
+            <UIKitLink target="_blank" href={iconPath}>
                 {this.renderIcon()}
-            </ExternalLink>
+            </UIKitLink>
         );
     }
     renderLink() {
         const {externalLink, name, path, onNameMouseEnter, onNameMouseLeave} = this.props;
 
         if (externalLink) {
-            return <ExternalLink href={path}>{name}</ExternalLink>;
+            return (
+                <UIKitLink className={b('name')} href={path}>
+                    {name}
+                </UIKitLink>
+            );
         }
 
         return path ? (
             <Link
+                className={b('name')}
                 title={name}
```

Comment:
an irritating nuance here
`title` is affected by `direction`, but not by `unicode-bidi`. So it displays with a reversed punctuation: “local/” instead of “/local”, etc.
The simplest solution is seems to be moving the title to the block root element. Not pretty, but works